docs/design-documents/hal/0004-pin-names-general-guidelines.md

```diff
@@ -104,20 +103,41 @@ Alternatively, if the usage of a BUTTON is required, then the application can de
         #error This application requires usage of a BUTTON
     #endif 
 
+It's possible to define new names of BUTTONs related its properties with corresponding comments. They can be defined as aliases at application level as shown below:
+
+    #define PUSH_BUTTON BUTTON1 // Momentary push Button
+
+However, these names do not apply to all boards and hence should not be used in official example applications that are considered platform agnostic.
+
+### UART
+
+Every Mbed board includes a serial interface to the host PC. This is a requirement and it's useful to be able to run automated tests using Greentea as well as displaying the status of the application.
```

Comment:
Every Mbed board includes a serial interface to the host PC, which allows console print useful for application status info o basic debugging ?
This is also a requirement to run automated... ?